tasks/eva_3779/ingest_remapped_vcf.nf

```diff
@@ -0,0 +1,48 @@
+
```

Comment:
Should we run the delete script as part of this nextflow, or will we just run it separately?

Reply:
I would prefer not to because the deletion will likely take a long time and therefore will need supervision.